core/domain/exp_domain.py

```diff
@@ -2331,6 +2331,31 @@ def _convert_states_v29_dict_to_v30_dict(cls, states_dict):
                 del answer_group['tagged_misconception_id']
         return states_dict
 
+    @classmethod
+    def _convert_states_v30_dict_to_v31_dict(cls, states_dict):
+        """Converts from version 30 to 31. Version 31 updates
+        the Voiceover duration to have duration of the audio file
+        under FileModel.
```

Comment:
`the audio file under FileModel.` I'm not sure about this can you please explain?

Reply:
Yeah should this make this more clear, this could be clarified as "Version 31 updates the Voiceover model to have an initialized duration attribute of 0.0. This will be updated when 
a new mp3 audio file is uploaded for the exploration." Sorry just leftover stuff from trying to figure out how the design doc was going to be written.

Reply:
done.